server/datastore/mysql/apple_mdm.go

```diff
@@ -3081,6 +3081,49 @@ WHERE h.uuid = ?
 	return nil
 }
 
+func (ds *Datastore) MDMAppleDDMSynchronizationTokens(ctx context.Context, teamID uint) (*fleet.MDMAppleDDMSyncTokens, error) {
+	const stmt = `
+SELECT
+	md5_checksum,
+	latest_created_timestamp
+FROM
+	team_declaration_checksum_view
+WHERE
+	team_id = ?
+`
+
+	var res fleet.MDMAppleDDMSyncTokens
+	if err := sqlx.GetContext(ctx, ds.reader(ctx), &res, stmt, teamID); err != nil {
+		return nil, ctxerr.Wrap(ctx, err, "get DDM checksum by team id")
+	}
+
+	return &res, nil
+}
+
+func (ds *Datastore) MDMAppleDDMDeclarationItems(ctx context.Context, teamID uint) ([]fleet.MDMAppleDDMDeclarationItemDB, error) {
+	// TODO: Confirm whether we can use JSON functions in the query (if 5.7 officially unsupported
+	// by Fleet)
```

Comment:
is this old? I don't see JSON usage. FWIW, I think we still need to support 5.7 for a while.

the issue to drop support is scheduled to happen, but until it's done, properly documented, etc let's assume 5.7 is still a requirement.

Reply:
edit: ah, I think this comment is so we don't have to loop over the results in the caller. Unfortunately, let's keep it as-is for now.